lib/liquid/registers/disabled_tags.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+module Liquid
+  class DisabledTags < Register
+    def initialize
+      @disabled_tags = Hash.new { |h, k| h[k] = 0 }
```

Comment:
Why keep a count and not true/false?

Reply:
To handle nested tags
```
Include (+1) > Render (=1) > Include (+1) > Render (=2)
```
Going back down this would be 
```
Render (=2) > Include (-1) > Render (=1) > Include (-1)
```
However if they were boolean
```
Include (true) > Render (=true) > Include (true) > Render (=true)
```
Going back down this would be 
```
Render (=true) > Include (false) > !!! Render (=false) !!!  > Include (false)
``` 
So in the last one the first include toggles it back but there is still a `render` below it